src/clusterfuzz/_internal/bot/tasks/update_task.py

```diff
@@ -194,8 +194,7 @@ def update_source_code():
   shell.clear_temp_directory()
 
   root_directory = environment.get_value('ROOT_DIR')
```

Comment:
Can we use the parent dir instead? Might be some weird issues with downloading the zip into the directory we're unzipping to.